.github/workflows/deploy-dev.yaml

```diff
@@ -0,0 +1,27 @@
+﻿name: Deploy to Dev
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: dev
+    env:
+      S3_URL: ${{ secrets.S3_URL }}

```

Comment:
S3? :)

Reply:
It's S3 Bucket Url
we have API Url and S3 Bucket Url

Reply:
It is used in this build
`run: yarn s3p sync -y -t $S3_URL`